### PR TITLE
[Validator] Updated documentation of URL validator

### PR DESCRIPTION
**Pull request by saro0h**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.7 |
| Fixed tickets | ~ |

Updated the documentation regarding the Pull Request on Symfony : https://github.com/symfony/symfony/pull/12956

### CODE REVIEW

**Comment by javiereguiluz** [comment]
I agree with @xabbuh: the first examples should not use the `checkDNS` option because its use case is too specific. Then you can explain it thoroughly in its own section.

**Comment by xabbuh** [comment]
I wish you a happy new year @saro0h. Can you update here given that the code pull request was merged?

File: reference/constraints/Url.rst [hunk 1]
```diff
@@ -26,6 +27,9 @@ Basic Usage
             properties:
                 bioUrl:
                     - Url: ~
+                        message: The url "{{ value }}" is not a valid url.
+                        protocols: [http, https]
+                        checkDNS: true
```
Comment (timglabisch) [inline reference/constraints/Url.rst]: would prefer if we set this to false because this is the default value.

**Comment by xabbuh** [comment]
@saro0h Can you update your pull request?

**Comment by WouterJ** [comment]
Hi @saro0h! Upcoming Saturday, we'll be hosting a [doc sprint day](http://symfony.com/blog/symfony-docs-hack-day-needs-you-on-may-23rd). We would like to finish as much PRs as possible during/before that day. Maybe you can join or, if you agree, we can put it on a list so others can work on it.

**Comment by javiereguiluz** [comment]
I'm finishing this PR in #5426.

@saro0h thanks for starting this pull request. In the new PR I've reused your commits to not lose the attribution of your work. Thanks!

**Comment by xabbuh** [comment]
Thank you for starting this @saro0h! Closing in favor of #5426.

**Pull request closed without merging**